testing/helpers.go

```diff
@@ -0,0 +1,88 @@
+package testing
```

Comment:
this should not be in a new package, these exported methods should be within the existing pkg/appliance, for example pkg/appliance/functions_tests.go where it's only being used.